sys/dev/drm/panfrost/panfrost_drv.c

```diff
@@ -342,15 +311,27 @@ panfrost_ioctl_submit(struct drm_device *dev, void *data,
 		}
 	}
 
+	pfile = file->driver_priv;
+
 	job = malloc(sizeof(*job), M_PANFROST1, M_WAITOK | M_ZERO);
 	job->sc = sc;
 	job->jc = args->jc;
 	job->requirements = args->requirements;
 	job->flush_id = panfrost_device_get_latest_flush_id(sc);
-	job->pfile = file->driver_priv;
+	job->mmu = pfile->mmu;
 
 	refcount_init(&job->refcount, 1);
 
+	slot = panfrost_job_get_slot(job);
+
+	job->slot = slot;
+
+	entity = &pfile->sched_entity[slot];
+
+	error = drm_sched_job_init(&job->base, entity, NULL);
+	if (error)
+		return (EINVAL);
```

Comment:
```suggestion
		return (-EINVAL);
```

Reply:
Why not just `return (error)`?

Reply:
That's true of all these... but maybe some of the panfrost_ ones are doing FreeBSD-style errors